### PR TITLE
Tidied up modules and renamed go-tidy to gotidy

### PR DESCRIPTION
**Pull request by owais**

### CODE REVIEW

**Comment by owais** [comment]
@tigrannajaryan I ran `go mod tidy` and this time it removed testbed from the main repo. I'm on Go 1.13.5.

File: Makefile [hunk 1]
```diff
@@ -36,8 +36,8 @@ test-with-cover:
 	 	go tool cover -html=coverage.txt -o coverage.html ); \
 	done
 
-.PHONY: go-tidy
-go-tidy:
+.PHONY: gotidy
```
Comment (tigrannajaryan) [inline Makefile]: One less character to type. I like it :-)

**Review by tigrannajaryan — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (owais) [reply Makefile]: :) also in line with other targets like `gofmt`

File: exporter/sapmexporter/go.mod [hunk 2]
```diff
@@ -4,7 +4,6 @@ go 1.12
 
 require (
 	github.com/open-telemetry/opentelemetry-collector v0.2.1-0.20191119152140-567e1046cefa
-	github.com/pierrec/lz4 v2.0.5+incompatible // indirect
```
Comment (pjanotti) [inline exporter/sapmexporter/go.mod]: Every time that I open repo using Goland it adds this back, I have to check my settings...

**Review by pjanotti — APPROVED** [review]
LGTM. I will be curious if someone has a Goland setup that doesn't keep adding `github.com/pierrec/lz4 v2.0.5+incompatible // indirect`

**Pull request merged**